### PR TITLE
fix: AttributeError: 'GoogleGemini' object has no attribute 'google'

### LINKED ISSUE #964: Unable to use agent.rephrase_query(query) or agent.clarification_questions(query) in using GoogleGemini
### System Info

OS: Linux
Python version: 3.10.13
pandasai.__version__ = '1.5.19' with google-ai extra

I am running the code on vertex ai notebook. I am able to use agent.chat but I am Unable to use agent.rephrase_query(query) or agent.clarification_questions(query) in using GoogleGemini

### 🐛 Describe the bug

sample code to reproduce the error.

``` Python

import pandas as pd
from pandasai import Agent
from pandasai.llm import GoogleGemini

employees_data = {
    "EmployeeID": [1, 2, 3, 4, 5],
    "Name": ["John", "Emma", "Liam", "Olivia", "William"],
    "Department": ["HR", "Sales", "IT", "Marketing", "Finance"],
}

salaries_data = {
    "EmployeeID": [1, 2, 3, 4, 5],
    "Salary": [5000, 6000, 4500, 7000, 5500],
}

employees_df = pd.DataFrame(employees_data)
salaries_df = pd.DataFrame(salaries_data)


llm = GoogleGemini(api_key=api_key)
agent = Agent([employees_df, salaries_df], config={"llm": llm}, memory_size=10)

query = "Who gets paid the most?"

# Chat with the agent
response = agent.chat(query)
print(f'{response} --- response from agent.chat')

# Get Clarification Questions
questions = agent.clarification_questions(query)

for question in questions:
    print(question)

# Explain how the chat response is generated
response = agent.explain()
print(response)

```

The highest salary is 7000. --- response from agent.chat
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[70], line 30
     27 print(f'{response} --- response from agent.chat')
     29 # Get Clarification Questions
---> 30 questions = agent.clarification_questions(query)
     32 for question in questions:
     33     print(question)

File /opt/conda/lib/python3.10/site-packages/pandasai/agent/__init__.py:137, in Agent.clarification_questions(self, query)
    128 """
    129 Generate clarification questions based on the data
    130 """
    131 prompt = ClarificationQuestionPrompt(
    132     dataframes=self._lake.dfs,
    133     conversation=self._lake._memory.get_conversation(),
    134     query=query,
    135 )
--> 137 result = self._call_llm_with_prompt(prompt)
    138 self._logger.log(
    139     f"""Clarification Questions:  {result}
    140     """
    141 )
    142 result = result.replace("```json", "").replace("```", "")

File /opt/conda/lib/python3.10/site-packages/pandasai/agent/__init__.py:67, in Agent._call_llm_with_prompt(self, prompt)
     65 while retry_count < self._lake.config.max_retries:
     66     try:
---> 67         result: str = self._lake.llm.call(prompt)
     68         if prompt.validate(result):
     69             return result

File /opt/conda/lib/python3.10/site-packages/pandasai/llm/base.py:543, in BaseGoogle.call(self, instruction, suffix)
    531 """
    532 Call the Google LLM.
    533 
   (...)
    540 
    541 """
    542 self.last_prompt = instruction.to_string() + suffix
--> 543 return self._generate_text(self.last_prompt)

File /opt/conda/lib/python3.10/site-packages/pandasai/llm/google_gemini.py:87, in GoogleGemini._generate_text(self, prompt)
     75 """
     76 Generates text for prompt.
     77 
   (...)
     83 
     84 """
     85 self._validate()
---> 87 completion = self.google.GoogleGemini.generate_content(
     88     contents=prompt,
     89     temperature=self.temperature,
     90     top_p=self.top_p,
     91     top_k=self.top_k,
     92     max_output_tokens=self.max_output_tokens,
     93 )
     95 return completion.text

AttributeError: 'GoogleGemini' object has no attribute 'google'

### LINKED ISSUE #995: AttributeError: 'GoogleGemini' object has no attribute 'google'
### System Info

OS version: windows11
Python version: 3.11
The current version of pandasai being used: 2.0.3

### 🐛 Describe the bug

```
import pandas as pd
from pandasai.llm import GooglePalm, GoogleGemini
from pandasai import Agent
df = pd.DataFrame({
    "country": [
        "United States",
        "United Kingdom",
        "France",
        "Germany",
        "Italy",
        "Spain",
        "Canada",
        "Australia",
        "Japan",
        "China",
    ],
    "gdp": [
        19294482071552,
        2891615567872,
        2411255037952,
        3435817336832,
        1745433788416,
        1181205135360,
        1607402389504,
        1490967855104,
        4380756541440,
        14631844184064,
    ],
    "happiness_index": [6.94, 7.16, 6.66, 7.07, 6.38, 6.4, 7.23, 7.22, 5.87, 5.12],
})
llm = GoogleGemini(api_key=API-KEY)
dfa = Agent([df], config={"llm": llm}, memory_size=10)

response = dfa.chat("Return the top 5 countries by GDP")
print(response)

response = dfa.chat("Plot the histogram of countries showing for each the gpd")
print(response)

```

--------------------------------------------------
Just a simple try with Gemini but with error:  the first chat got success response, but the second(to plot a chart) got error. See below trace:

'''
          country             gdp  happiness_index
0   United States  19294482071552             6.94
9           China  14631844184064             5.12
8           Japan   4380756541440             5.87
3         Germany   3435817336832             7.07
1  United Kingdom   2891615567872             7.16
Traceback (most recent call last):
  File "C:\Users\Danzel\PycharmProjects\Tools\venv\Lib\site-packages\pandasai\pipelines\chat\generate_chat_pipeline.py", line 155, in run
    output = self.pipeline.run(input)
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Danzel\PycharmProjects\Tools\venv\Lib\site-packages\pandasai\pipelines\pipeline.py", line 137, in run
    raise e
  File "C:\Users\Danzel\PycharmProjects\Tools\venv\Lib\site-packages\pandasai\pipelines\pipeline.py", line 101, in run
    step_output = logic.execute(
                  ^^^^^^^^^^^^^^
  File "C:\Users\Danzel\PycharmProjects\Tools\venv\Lib\site-packages\pandasai\pipelines\chat\code_generator.py", line 33, in execute
    code = pipeline_context.config.llm.generate_code(input, pipeline_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Danzel\PycharmProjects\Tools\venv\Lib\site-packages\pandasai\llm\base.py", line 196, in generate_code
    response = self.call(instruction, context)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Danzel\PycharmProjects\Tools\venv\Lib\site-packages\pandasai\llm\base.py", line 483, in call
    return self._generate_text(self.last_prompt, memory)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Danzel\PycharmProjects\Tools\venv\Lib\site-packages\pandasai\llm\google_gemini.py", line 90, in _generate_text
    completion = self.google.GoogleGemini.generate_content(
                 ^^^^^^^^^^^
AttributeError: 'GoogleGemini' object has no attribute 'google'
Unfortunately, I was not able to answer your question, because of the following error:

'GoogleGemini' object has no attribute 'google'

'''

### PR DESCRIPTION
**Pull request by traien**
- [x] Closes #964.
- [ ] Closes #995 .
- [x] Tests added and passed.
- [ ] All [code checks passed](https://github.com/gventuri/pandas-ai/blob/main/CONTRIBUTING.md#-testing).

### CODE REVIEW

File: pandasai/llm/google_gemini.py [hunk 1]
```diff
@@ -87,12 +87,8 @@ def _generate_text(self, prompt: str, memory: Optional[Memory] = None) -> str:
         updated_prompt = self.prepend_system_prompt(prompt, memory)
 
         self.last_prompt = updated_prompt
-        completion = self.google.GoogleGemini.generate_content(
+        completion = self.google_gemini.generate_content(
             contents=prompt,
-            temperature=self.temperature,
-            top_p=self.top_p,
-            top_k=self.top_k,
-            max_output_tokens=self.max_output_tokens,
```
Comment (gventuri) [inline pandasai/llm/google_gemini.py:95]: Why did you remove these params?

**Review by gventuri — COMMENTED** [review]
Looks good, only one comment! @traien

File: pandasai/llm/google_gemini.py (hunk 1 above)
Comment (traien) [reply pandasai/llm/google_gemini.py:95]: I got this error:
File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/pandasai/llm/google_gemini.py", line 91, in _generate_text
    completion = self.google_gemini.generate_content(
  File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/google/generativeai/generative_models.py", line 234, in generate_content
    request = self._prepare_request(
  File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/google/generativeai/generative_models.py", line 215, in _prepare_request
    return glm.GenerateContentRequest(
  File "/Users/osama/PycharmProjects/testPandasAI/.venv/lib/python3.10/site-packages/proto/message.py", line 576, in __init__
    raise ValueError(
ValueError: Unknown field for GenerateContentRequest: temperature


but I think we can solve it without removing params, will do it on the next commit

Comment (gventuri) [reply pandasai/llm/google_gemini.py:95]: @traien thanks a lot, merging!
